### PR TITLE
Added a module that displays where the tab is currently rendered.

### PR DESCRIPTION
**Pull request by czack810150**

### CODE REVIEW

File: package/manifest.json [hunk 1]
```diff
@@ -41,8 +41,8 @@
         "groupchat"
       ],
       "meetingSurfaces": [
-        "Stage",
-        "SidePanel"
+        "stage",
```
Comment (czack810150) [inline package/manifest.json:44]: I changed these to camelCase because the Teams is complaining it and the build wouldn't succeed if I don't change it...

Comment (czack810150) [reply package/manifest.json:44]: I also think the meetingSurfaces is replaced with context according to the [manifest schema reference](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema). Please let me know if I should update it.

Comment (sachingupta) [reply package/manifest.json:44]: Revert these change. since these are values we use.

File: src/app.ts [hunk 2]
```diff
@@ -8,6 +8,12 @@ export const initializeAppModules = () => {
     microsoftTeams.initialize();
     microsoftTeams.appInitialization.notifyAppLoaded();
 
+    if (isInTeams()){
+      microsoftTeams.getContext(outputTabRenderedLocation);
+    }else{
```
Comment (sachingupta) [inline src/app.ts:13]: Move everything under this function outputTabRenderedLocation

**Review by sachingupta — APPROVED** [review]

**Comment by czack810150** [comment]
@sachingupta Could you review again?

**Review by sachingupta — APPROVED** [review]

**Pull request merged**